File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikSectionsAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.sections.StepikSectionsGetQuery;

/**
 * @author meanmail
 */
public class StepikSectionsAction extends StepikAbstractAction {
    public StepikSectionsAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikSectionsGetQuery get() {
        return new StepikSectionsGetQuery(this);
    }
}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikSectionsAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.sections.StepikSectionsGetQuery

class StepikSectionsAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun get(): StepikSectionsGetQuery {
        return StepikSectionsGetQuery(this)
    }
}
